packages/main/src/Button.hbs

```diff
@@ -16,9 +16,11 @@
 			></ui5-icon>
 		{{/if}}
 
-		{{#if ctr._nodeText}}
+		{{#if ctr.text.length}}
```

Comment:
why not always render the slot? Also, earlier the check was for non-empty text, now it can be an array with an empty text and the check will match.